### DISCUSSION TITLE
Projects were paused from exceeding Vercel Function execution limits

### DISCUSSION THREAD

**Discussion by odysseaspapadimas** [vercel/community#7278] [Account Issue]
### Question

Hello, I have honestly stopped paying attention to my projects for some time now as they had been completely inactive and to my suprise I found out that they'd been paused due to excessive usage. This is clearly some type of DDoS attack or something along those lines because you can see I started receiving hundreds of thousands of requests.

What would be required in order to unpause my projects? I would obviously pause the project myself if I ever notice unusual activity like this again, I just never expected it since I never share my projects

**Comment by leerob** [comment]
Hey there. Could you please open a support request so we can look at your paused free account?

In the future, you can set up [custom rules for the Vercel Firewall](https://youtu.be/t-U6vTrx_9Y) to prevent against malicious behavior as well.

**Comment by odysseaspapadimas** [comment]
@leerob Thanks for the quick reply, I submitted a case under the Usage Enquiry problem area

**Comment by odysseaspapadimas** [comment]
This was resolved, thank you

[a comment by an automated account is not shown]
